admin_interface/templatetags/admin_interface_tags.py

```diff
@@ -103,9 +103,19 @@ def get_admin_interface_nocache():
     return hash_string(__version__)
 
 
-@simple_tag()
-def admin_interface_clear_filter_qs(changelist, list_filter):
-    return changelist.get_query_string(remove=list_filter.expected_parameters())
+@simple_tag(takes_context=False)
+def get_admin_interface_active_date_hierarchy(changelist):
```

Comment:
Could you add a basic test for this template tag?

Reply:
Should I use stubs for changelist and list_filter (making tests code very dependent on tested code), or configure a test site and use django-webtest to send real requests?

Reply:
If you can/want to use `django-webstest` it would be great and it would also be a good start for testing next UI improvements.